crates/sqlexec/src/planner/logical_plan.rs

```diff
@@ -67,6 +67,25 @@ impl LogicalPlan {
     /// Note this currently only replaces placeholders for datafusion plans.
     pub fn replace_placeholders(&mut self, scalars: Vec<ScalarValue>) -> Result<()> {
         if let LogicalPlan::Query(plan) = self {
+            // Replace placeholders in the inner plan if the wrapped in an
+            // EXPLAIN.
+            //
+            // TODO: Make sure this is the correct behavior.
```

Comment:
From some initial testing, it _does_ seem like postgres handles this case, but I'm getting some weird output when using pgprototest:

```
send
Parse {"query": "explain select $1"}
Bind {"values": ["4"]}
Execute
Sync
----
----

----
---- (no newline)

until
ReadyForQuery
----
ParseComplete 
BindComplete 
DataRow {"fields":["Result  (cost=0.00..0.01 rows=1 width=32)"]}
CommandComplete {"tag":"EXPLAIN"}
ReadyForQuery {"status":"I"}
```

Reply:
Actually the extra lines just seem like a regression on our end.